locales/en/messages.json

```diff
@@ -7738,6 +7738,9 @@
     "firmwareFlasherOptionLabelLegacy": {
         "message": "Legacy"
     },
+    "firmwareFlasherOptionLabelNotQualified": {
+        "message": "Non qualified target — flash at your own risk - not officially supported"
```

Comment:
This is interesting; However,  the verbiage `Non qualified target` might reflect wrongly as "low-quality" or "substandard" (even though potentially true).

I rather see:
`Officially Supported Target - Verified Partner`
and
`Target not a Verified Partner product`
or something similar, not necessarily offering quality indicator.

i.e. nothing "wrong" with a dozen of my FC's that are not by Verified Partners.

Thoughts @sugaarK ?